support_override_styles/src/main/java/bfahy/support_override_styles/Global.kt

```diff
@@ -10,9 +10,9 @@ import zendesk.support.Support
 
 class Global : Application() {
 
-    private val SUBDOMAIN_URL = ""
-    private val APPLICATION_ID = ""
-    private val OAUTH_CLIENT_ID = ""
+    private val SUBDOMAIN_URL = "https://z3nadvocate84.zendesk.com"
```

Comment:
should we provide these credentials here?

Reply:
I forgot to remove, will remove that